bin/picca_Pk1D.py

```diff
@@ -70,7 +70,24 @@ def process_all_files(index_file_args):
     # read fits or ascii file
     if args.in_format == 'fits':
         hdul = fitsio.FITS(file)
-        deltas = [Delta.from_fitsio(hdu, pk1d_type=True) for hdu in hdul[1:]]
+        try:
+            deltas = [Delta.from_fitsio(hdu, pk1d_type=True) for hdu in hdul[1:]]
+            running_on_raw_transmission = False
+        except ValueError:
+            print("\nPk1d_type=True didn't work on read in, maybe perfect model? Trying without!")
+            deltas = [delta.from_fitsio(hdu,Pk1D_type=False) for hdu in hdul[1:]]
```

Comment:
I'm testing this and I have the following error at this line:
NameError: free variable 'delta' referenced before assignment in enclosing scope
Maybe you do not need a from_fitsio function here ?